### DISCUSSION TITLE
[Projects beta]: Ability to view achieved cards/issues

### DISCUSSION THREAD

**Discussion by andre-wells** [community/community#8211] [Projects and Issues] [answered]
In the current Project Boards, there is an easy way to [view archived items and to restore them](https://docs.github.com/en/issues/organizing-your-work-with-project-boards/tracking-work-with-project-boards/archiving-cards-on-a-project-board).

But there's no obvious way to do the same thing in Projects (beta).  Can a menu item be added next to Workflows and Settings?

![image](https://user-images.githubusercontent.com/66315105/143864619-e6757112-3b0d-47cd-bfd5-cd7a68232b05.png)

The only way I can see to restore an item is via the issue itself.

![image](https://user-images.githubusercontent.com/66315105/143866739-541ef1ec-5de1-451b-a341-18c2097cfb7e.png)

**Comment by mobile-sergey** [comment, marked as answer]
**Dubllicate**: [#7501](https://github.com/github/feedback/discussions/7501) Projects (Beta) - View Archived cards
